### ISSUE TITLE
Strong named assembly

### ISSUE THREAD

**Issue by lauxjpn** [tomaszeman/Xunit.Extensions.Ordering#6]
We need this to be a strong named assembly to use in your strong named assembly project.

**Comment by lauxjpn** [comment]
I assume this repo got abandoned.

**Comment by GeorgDangl** [comment]
Hey @lauxjpn, I've created a fork, the assembly now has a strong name:
https://www.nuget.org/packages/Dangl.Xunit.Extensions.Ordering

**Comment by lauxjpn** [comment]
@GeorgDangl Nice one!

We implemented some [simple extensions](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/tree/master/test/EFCore.MySql.FunctionalTests/TestUtilities/Xunit) for our xUnit scenarios some time ago.

**Comment by GeorgDangl** [comment]
Ah, not bad😀 We we also using it for Docker based SQL testing, so we didn’t use the orderer but the assembly fixture.

**Comment by tomaszeman** [comment]
> Ah, not bad😀 We we also using it for Docker based SQL testing, so we didn’t use the orderer but the assembly fixture.

Yes, Assembly fixture was only valuable part of implementation (and only proper that time :-) As I stated in documentation, test ordering for any purpose is bad practice (eclipse of the mind :-). I wanted to implement test dependency what only makes some sense for using xunit for integration testing but it would require quite much time and hacking bcs. of some "odd" design decisions in xunit core.

**Comment by lauxjpn** [comment]
We enable explicit ordering for specific debugging cases, when some unintended side effects between test cases seem to exists (e.g. one test case fails if a particular test case ran before). That allows us to narrow down such issues quickly.

We have also used it from time to time to diff the test outputs.
